incipyt/_internal/dumpers.py

```diff
@@ -0,0 +1,78 @@
+from string import Formatter
+
+import collections.abc
+import configparser
+import pathlib
+import toml
+
+
+class BaseDumper(type(pathlib.Path())):
+    @classmethod
+    def make(cls, path, sanitizer=None):
+        self = cls(path)
+        self._sanitizer = sanitizer
+        self._root = None
+
+        return self
```

Comment:
Any reason this classmethod is not an initializer?
```suggestion
    def __init__(self, path, sanitizer=None):
        super().__init__(path)
        self._sanitizer = sanitizer
        self._root = None
```

Reply:
As `path.Path` is a bit magic, it don't work if you do that.